wallet/Cargo.toml

```diff
@@ -8,7 +8,10 @@ authors = ["Ignotus Peverell <igno_peverell@protonmail.com>", "Laurent Meunier <
 byteorder = "1"
 log = "^0.3"
 rand = "^0.3"
-rust-crypto = "^0.2"
+ripemd160 = "~0.6.0"
+# TODO - use the crate once this get merged in (assuming it does)
+# sodiumoxide = "~0.0.15"
+sodiumoxide = { git = "https://github.com/oherrala/sodiumoxide", branch = "compile-from-git" }
```

Comment:
We probably want to either fork the official sodiumoxide repo into mimblewimble ourselves and manage it there or wait for the related PR to be merged in.
https://github.com/dnaq/sodiumoxide/pull/167